packages/react-aria-components/src/Popover.tsx

```diff
@@ -59,7 +59,16 @@ function Popover(props: PopoverProps, ref: ForwardedRef<HTMLElement>) {
   let isExiting = useExitAnimation(ref, state.isOpen);
 
   if (state && !state.isOpen && !isExiting) {
-    return preserveChildren ? <HiddenContext.Provider value>{props.children}</HiddenContext.Provider> : null;
+    let children = props.children;
+    if (typeof children === 'function') {
+      children = children({
+        placement: 'bottom',
```

Comment:
Why is this placement not user defined?

Reply:
Because in this case the popover isn't actually rendering, it's just passing through its children. But we have to have _something_ to pass to the function, so I just chose one. It doesn't actually matter what the value is.